Automattic/sensei
# Implement Course Theme Lesson|Module Title blocks.

Fixes #4448 

### Changes proposed in this Pull Request

* Implements `sensei-lms/course-theme-module-title`.
* Implements `sensei-lms/course-theme-lesson-title`.

### Testing instructions


* Enable the feature flag: `add_filter( 'sensei_feature_flag_course_theme', '__return_true' );`
* Confirm module title and lesson title are correctly rendered on Course Theme lesson content area.


### Screenshot / Video


https://user-images.githubusercontent.com/2578542/145582136-692f9ed8-b987-4f52-bcdd-527f2fb203a4.mp4
